src/Components/Pages/Results/Results.scss

```diff
@@ -32,22 +30,21 @@ $eligibilityTablePaddingH: 36px;
     }
   }
 
-  .eligibility__table,
+  .criteria-table,
   .protections-on-next-page__print {
-    border-radius: 4px;
-    border: 2px solid $GREY_NEW;
-    padding: 1.5rem $eligibilityTablePaddingH; // 24px
+    // border-radius: 4px;
```

Comment:
unsure if you wanted to leave in this commented out code

Reply:
thanks - meant to delete all that